x/rollapp/client/cli/tx_create_rollapp.go

```diff
@@ -14,27 +14,30 @@ import (
 
 func CmdCreateRollapp() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "create-rollapp [rollapp-id] [alias] [vm-type] [init-sequencer-address] [metadata]",
+		Use:     "create-rollapp [rollapp-id] [alias] [bech32-prefix] [vm-type] [init-sequencer-address] [genesis_checksum] [metadata]",
 		Short:   "Create a new rollapp",
 		Example: "dymd tx rollapp create-rollapp ROLLAPP_CHAIN_ID Rollapp ethm EVM '<seq_address1>,<seq_address2>' <genesis_checksum> metadata.json",
 		Args:    cobra.MinimumNArgs(4),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			// nolint:gofumpt
-			argRollappId, alias, vmTypeStr := args[0], args[1], args[2]
+			argRollappId, alias, argBech32Prefix, vmTypeStr := args[0], args[1], args[2], args[3]
```

Comment:
bech32 shouldn't be required on rollapp creation.
should be rollapp-id, alias, vm-type. the rest optional.

Reply:
you right, fixed